### ISSUE TITLE
GL ERROR 65539: Invalid key -1

### ISSUE THREAD

**Issue by Hyomoto** [Luke100000/ImmersiveAircraft#19]
As of the latest update, whenever I'm inside of an aircraft, my log is spammed with the provided error.  Full error is:
```
[19:07:47] [Render thread/ERROR]: ########## GL ERROR ##########
 [19:07:47] [Render thread/ERROR]: @ Render 
[19:07:47] [Render thread/ERROR]: 65539: Invalid key -1
```
Full log is here:
<https://mclo.gs/nKjj3GV>

Wouldn't surprise me if it's a conflict, but this wasn't happening on the prior version as far as I noticed.

**Comment by Luke100000** [comment]
Huh. Now since I still need to hack around with custom keybinds so I have multiple bindings on one key. I used the unknown key, -1 to prevent Minecraft to interfere with my mechanics.

I think it's partially a mod conflict but I will see how to properly suppress that warning. -1 is also used for unused keys so I do not fully understand the issue yet. I could use a dummy key, but that would occupy said key, e.g. F20 or something. Thanks for the report.

**Comment by Onewingseraphim** [comment]
Any possibility to fix this for 1.19.2?
